src/System.Device.Gpio.Tests/GpioControllerTestBase.cs

```diff
@@ -270,16 +270,11 @@ void Callback(object sender, PinValueChangedEventArgs e)
             }
         }
 
-        [Fact]
+        // [Fact] // https://github.com/dotnet/iot/issues/629
+        // Test seem to be failing with almost all drivers
         public void AddCallbackRemoveAllCallbackTest()
```

Comment:
How did you determine which test to disable? I was going to do some Kusto queries when I got back in order to do some analysis to figure out which tests were the problematic ones and on which platforms did they fail

Reply:
it keeps on failing on CI, I've never used Kusto queries, if you know an easy way to determine all failing tests that would be useful